src/platform/user/profile/vet360/components/base/VAPEditView.jsx

```diff
@@ -110,20 +110,22 @@ class VAPEditView extends Component {
         isLoading={isLoading}
         deleteEnabled={!isEmpty && !deleteDisabled}
       >
-        <LoadingButton
-          data-action="save-edit"
-          isLoading={isLoading}
-          className="vads-u-width--auto vads-u-margin-top--0"
-        >
-          Update
-        </LoadingButton>
-        <button
-          type="button"
-          className="usa-button-secondary medium-screen:vads-u-margin-left--2 vads-u-margin-top--0 vads-u-width--auto"
-          onClick={onCancel}
-        >
-          Cancel
-        </button>
+        <div>
+          <LoadingButton
```

Comment:
I noticed that the visual bug you are fixing showed up when this `LoadingButton` was showing the spinner since its width shrunk down a whole lot.

Reply:
Gotcha! The `flex-direction: column` should fix that 😄